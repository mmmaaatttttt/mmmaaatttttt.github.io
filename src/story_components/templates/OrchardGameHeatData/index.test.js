import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { OrchardGameHeatData } from ".";

describe("smoke and snapshot tests", () => {
  it("renders successfully", () => {
    shallow(<OrchardGameHeatData />);
  });

  it("renders first demonstration successfully", () => {
    const wrapper = shallow(<OrchardGameHeatData />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});