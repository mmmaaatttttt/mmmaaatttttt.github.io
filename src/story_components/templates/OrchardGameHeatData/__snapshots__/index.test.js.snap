// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`smoke and snapshot tests renders first demonstration successfully 1`] = `
<SliderProvider
  column={true}
  fullWidthAt="small"
  initialData={
    Array [
      Object {
        "color": "#555555",
        "initialValue": 4,
        "max": 5,
        "min": 1,
        "step": 1,
        "tickCount": 5,
        "title": [Function],
      },
      Object {
        "color": "#555555",
        "initialValue": 1,
        "max": 5,
        "min": 1,
        "step": 1,
        "tickCount": 5,
        "title": [Function],
      },
    ]
  }
  render={[Function]}
  width="50%"
/>
`;
